src/imageio/storage/disk.c

```diff
@@ -201,8 +202,9 @@ void gui_init(dt_imageio_module_storage_t *self)
   d->onsave_action = dt_bauhaus_combobox_new(NULL);
   dt_bauhaus_widget_set_label(d->onsave_action, NULL, N_("on conflict"));
   dt_bauhaus_combobox_add(d->onsave_action, _("create unique filename"));
-  dt_bauhaus_combobox_add(d->onsave_action, _("overwrite"));
+  dt_bauhaus_combobox_add(d->onsave_action, _("prompt"));
```

Comment:
"prompt" isn't descriptive enough - it doesn't say what it actually does - and there's no tooltip to provide a clue.